spec/swagger.yaml

```diff
@@ -1649,6 +1649,51 @@ paths:
         429: *Error429ResponseDefinition
         500: *Error500ResponseDefinition
 
+  '/published-files-api/v2/projects/{projectId}/files/list/recently-published':
+    get:
+      summary: List recently published files in project
+      description: |
+        Return list of files that were published in last 14 days
```

Comment:
If a string is republished, the file would appear here again, right. Any good way to note that.